test/multiSamlStrategy.spec.ts

```diff
@@ -20,7 +21,7 @@ describe("MultiSamlStrategy()", function () {
       return { cert: FAKE_CERT };
     }
     const strategy = new MultiSamlStrategy({ getSamlOptions }, noop);
-    strategy.should.be.an.instanceOf(SamlStrategy);
+    strategy.should.be.an.instanceOf(AbstractStrategy);
```

Comment:
We should probably include another assert to make sure that it also extends `PassportStrategy`, just to make it clear the chain goes back to something that works with Passport.